packages/logs/src/domain/loggerSession.ts

```diff
@@ -29,7 +29,7 @@ export function startLoggerSession(configuration: Configuration, areCookieAuthor
   }
 }
 
-function computeTrackingType(configuration: Configuration): string {
+function computeTrackingType(configuration: Configuration): LoggerTrackingType {
```

Comment:
```suggestion
function computeTrackingType(configuration: Configuration) {
```
the compiler should figure it out, explicit typing does not seem necessary here